datasets/us/de/med_exclusions/crawler.py

```diff
@@ -41,37 +41,51 @@ def crawl_item(row: Dict[str, str], context: Context):
     entity.add("name", name)
     entity.add("alias", alias)
     entity.add("country", "us")
+    entity.add("sector", row.pop("taxonomy"))
+    entity.add("idNumber", row.pop("dea"))
 
     npi = row.pop("npi")
-        if npi != "N/A":
-            entity.add(
+    if npi != "N/A":
+        entity.add(
             "npiCode",
             h.multi_split(
                 npi,
                 [";", ",", "&"],
             ),
         )
 
-    if row.get("license") != "N/A":
-        entity.add("idNumber", row.pop("license").split(","))
-    else:
-        row.pop("license")
+    license_number = row.pop("license")
+
+    if license_number != "N/A":
+        entity.add("idNumber", license_number.split(","))
 
     sanction = h.make_sanction(context, entity)
     h.apply_date(sanction, "startDate", row.pop("effective_date"))
     sanction.add("description", row.pop("comments"))
     sanction.set("authority", row.pop("oig_medicaid_sanction"))
 
-h.apply_date(sanction, "endDate", row.pop("reinstated_date"))
-end_date = max(sanction.get('endDate'), None)
-if end_date is not None: 
-      target = datetime.strptime(reinstatement_date, "%Y-%m-%d") >= settings.RUN_TIME_ISO
-else:
-         target = True
+    reinstated_date = row.pop("reinstated_date")
+    h.apply_date(sanction, "endDate", reinstated_date)
+
+    end_date = sanction.get("endDate")
+    if end_date and reinstated_date not in [
```

Comment:
There are some reinstated dates that are like Suspended, Rescinded, Annulled. In this cases, I understand that the entity is not a target anymore. Correct? What I did was add that condition to the if statement.

Reply:
Since you've already set all these values to be `null`  in the lookups:  
```
        "Revoked",
        "Annulled",
        "Suspended",
        "Rescinded",
        "Preclusion"
```
this check `if end_date and reinstated_date not in [...]` won't make any difference. We can check it by printing both of them: 

```
    h.apply_date(sanction, "endDate", reinstated_date)
    print(reinstated_date)

    end_date = sanction.get("endDate")
    print(end_date)
```
we'll get: 
```
Indefinite
[]
Indefinite
[]
```
`end_date` is a processed `reinstated_date`. So, looking them up in the `yml` is enough

Reply:
Suspended seems to be used in the comments to mean they are suspended from providing services. Preclusion seems to mean they are precluded from providing services. Probation seems to be the period during which they may not participate.

Rescinded is used in the comments to mean their sanction was rescinded - they are free to provide services.

So I think the only one to consider a target is rescinded